Packs/SecureWorks/ReleaseNotes/4_0_0.json

```diff
@@ -0,0 +1 @@
+{"breakingChanges":true,"breakingChangesNotes":"Version 1 of the Alerts API has been decommissioned and replaced with Alerts v2. This migration means the fields in the alerts now differ from the original fields. Additionally, Investigations no longer use the `alerts` field, but now contain `alerts2`. Any references to the Alerts v1 fields will fail."}
```

Comment:
Please add the context change here as well, as this will show in the Breaking Change warning.